esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/adding_a_recipe.rst

```diff
@@ -0,0 +1,81 @@
+Adding a recipe to the Recipe Test Workflow (|RTW|)
```

Comment:
Would it be possible for the title to be "How to add a recipe to the Recipe Test Workflow (|RTW|)", please? 😊

Reply:
Amended in commit d01a94f42